packages/fast-components-class-name-contracts-base/src/Slider/index.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * The class name contract for the slider component
+ */
+export interface SliderClassNameContract {
+    /**
+     * The root of the slider component
+     */
+    slider?: string;
+
+    /**
+     * The layout panel of the slider component. Slider children end up in this container.
+     */
+    slider_layoutPanel?: string;
+
+    /**
+     * The fixed background track of the slider component
+     * which determines click region and extent of the slider
+     */
+    slider_barTrack?: string;
+
+    /**
+     * The cosmetic background track of the slider component
+     */
+    slider_barBack?: string;
+
+    /**
+     * The cosmetic foreground bar of the slider component that tracks the thumb
+     */
+    slider_barFront?: string;
+
+    /**
+     * The horizontal orientation modifier
+     */
+    slider__orientationHorizontal?: string;
+
+    /**
+     * The vertical orientation modifier
+     */
+    slider__orientationVertical?: string;
+
+    /**
+     * The base thumb class
+     */
+    slider_thumb?: string;
+
+    /**
+     * The low value thumb
+     */
+    slider_thumb_upper?: string;
```

Comment:
no multiple elements in BEM, only a single block, element or modifier is allowed, so this should be something like `slider_thumb__upperValue`

Reply:
yep